src/Provider/FacebookUser.php

```diff
@@ -171,6 +171,28 @@ public function getTimezone()
     }
 
     /**
+     * Returns the lower bound of the user's age range
+     *
+     * @return integer|null
+     */
+    public function getMinAge()
+    {
+        if (array_key_exists('min', $this->getField('age_range')))
```

Comment:
This `if` statement needs curly brackets. Also add `return null;` after closing `}`. Same goes for `getMaxAge()` below. 

Also this might be better written as:

```php
if (isset($this->data['age_range']['min'])) {
```